pkg/cluster/ansible/dirs.go

```diff
@@ -123,6 +125,21 @@ func parseDirs(user string, ins spec.InstanceSpec, sshTimeout int64) (spec.Insta
 				newIns.DeployDir = strings.Trim(strings.Split(line, " ")[1], "\"")
 				continue
 			}
+
```

Comment:
It's better to add some tests.

Reply:
forgot to commit the file: https://github.com/pingcap/tiup/pull/612/commits/6dd6f95c1dd479b206e26e8982d2ad8476cd73c7
but it only covers about parsing the file and get info